modin/core/dataframe/pandas/partitioning/partition.py

```diff
@@ -427,6 +427,23 @@ def empty(cls):
         """
         return cls.put(pandas.DataFrame(), 0, 0)
 
+    @classmethod
+    def partition_from_data(cls, df):
```

Comment:
It's pretty clear to use the old `put` method I think.

Reply:
Added this for verbosity , will remove.